avalon/manage.py

```diff
@@ -1,7 +1,9 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python2
 import os
 import sys
 
+sys.path.append('/home/paderball/projects/django')
```

Comment:
... not sure why you needed this. Something weird with the way you installed Django, I guess? Anyway, just don't include your `manage.py` changes in your final changes.